EventPlanning.BL/EventManager.cs

```diff
@@ -36,7 +36,7 @@ public List<Event> GetUserEvents(int userId)
         {
             try
             {
-                return repository.GetEvents().Where(e => e.UserId == userId).ToList();
+                return repository.GetEvents()?.Where(e => e.UserId == userId).ToList();
             }
             catch (ArgumentNullException)
```

Comment:
After the change is it possible to catch this exception? What is the point of catching exception if there is no logging?